src/Tezos/TWTezosForger.cpp

```diff
@@ -109,7 +96,8 @@ std::string forgeAddress(const std::string address) {
     result += TW::hex(decoded, decoded + decodedLength);
     result += "00";
   } else {
-    result = result + "00";
+    // tz1 address
```

Comment:
This could be a tz1 / tz2 / tz3.

The correct term would be `implicit address` or `public key hash`